README.md

```diff
@@ -130,7 +270,17 @@ tmp/firebase-export-metadata.json
 tmp/firestore_export
 ```
 
-### Stripe
+---
+<!-- section 3 -->
+
+## Part Three: Configure Third-Party Integrations
+
+### Optional: Twilio
+
+TODO: Add steps.
+
+
+### Optional: Stripe
```

Comment:
Ok... got it and stripe was causing some issues anyway. Worth trying to clean up main code base and rid it of Stripe completely? Since we don’t really have a working implementation yet... to my knowledge

Reply:
Not in time for this event. It’s a known tech debt thing, nothing needs to be done for setup, it will make some slight noise in the console. Far higher priority things to allocate time to, zero risk for not cleaning it up now.